File: Assignment2/StudentManagement/StudentManagement/Controllers/HomeController.cs
using System.Diagnostics;
using Microsoft.AspNetCore.Mvc;
using StudentManagement.Models;

namespace StudentManagement.Controllers;

public class HomeController : Controller {
    private readonly ILogger<HomeController> _logger;

    public HomeController(ILogger<HomeController> logger)
    {
        _logger = logger;
    }

    public IActionResult Index()
    {
        ViewData["Message"] = "Welcome to ASP.NET Core MVC";
        return View();
    }

    public IActionResult About()
    {
        ViewBag.StudentName = "Nhat Minh Quang";
        return View();
    }
    
    public IActionResult Contact()
    {
        ViewBag.Mail = "BIT24@st.cmcu.edu.vn";
        return View();
    }
    public IActionResult Privacy()
    {
        return View();
    }

    [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
    public IActionResult Error()
    {
        return View(new ErrorViewModel { RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier });
    }
}

File: JavaSB_Assignment/SpringBoot/src/main/java/org/example/springboot/controller/HomeController.java
package org.example.springboot.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;

@Controller
@RequestMapping("/home")
public class HomeController {
    @GetMapping({"", "/", "/index"})
    @ResponseBody
    public String index() { return "Welcome to Spring Boot"; }

    @GetMapping("/about")
    @ResponseBody
    public String about() { return "Nhat Minh Quang"; }

    @GetMapping("/contact")
    @ResponseBody
    public String contact() { return "BIT240199@st.cmcu.edu.vn"; }
}
